.vscode/c_cpp_properties.json

```diff
@@ -7,7 +7,7 @@
         "${workspaceFolder}/node_modules/node-addon-api",
         "${default}"
       ],
-      "defines": ["_DEBUG", "WINDOWS"],
+      "defines": ["_DEBUG", "MACOS"],
```

Comment:
Interesting, I wonder if we can make this dependent on the OS VS Code runs on.

Reply:
It appears there is, I just missed it was already platform-specific config
https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference